index.html

```diff
@@ -225,7 +225,8 @@ <h5 class="grey-text text-lighten-4 an-1 ">
                         <div class="footer-copyright">
                             <div class="container swiper-no-swiping">
                                 <div>
-                                    Copyright 2019 <a class="blue-text text-lighten-3 text-underLine"
+                                   Copyright <span class="an-1">{{ new Date().getFullYear() }}</span>
```

Comment:
Thank for your contribution. 

I think the single year copy right should state the year which the first time it published. I think this should not be dynamic

Reply:
what about having something like 2019-current year?

Reply:
That should work.

Reply:
Ok let me make that change then.